components/derive/src/exonum_interface.rs

```diff
@@ -164,17 +167,39 @@ impl TryFrom<&[Attribute]> for ExonumServiceAttrs {
     }
 }
 
+#[derive(Debug, FromMeta)]
+struct MethodIdAttr {
```

Comment:
Nit: Shouldn't this type be named `InterfaceMethodAttrs` or something like that?